[2022-11-20T07:00:14.611+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: python_dependences.task3 scheduled__2022-11-19T00:00:00+00:00 [queued]>
[2022-11-20T07:00:14.720+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: python_dependences.task3 scheduled__2022-11-19T00:00:00+00:00 [queued]>
[2022-11-20T07:00:14.724+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-20T07:00:14.728+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 3
[2022-11-20T07:00:14.740+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-20T07:00:15.192+0000] {local_task_job.py:87} ERROR - Received SIGTERM. Terminating subprocesses
[2022-11-20T07:00:15.216+0000] {local_task_job.py:164} INFO - Task exited with return code 143
[2022-11-20T07:00:19.751+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-11-21T10:40:35.609+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: python_dependences.task3 scheduled__2022-11-19T00:00:00+00:00 [queued]>
[2022-11-21T10:40:35.722+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: python_dependences.task3 scheduled__2022-11-19T00:00:00+00:00 [queued]>
[2022-11-21T10:40:35.729+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-21T10:40:35.731+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 3
[2022-11-21T10:40:35.732+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-21T10:40:35.858+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): task3> on 2022-11-19 00:00:00+00:00
[2022-11-21T10:40:35.992+0000] {standard_task_runner.py:55} INFO - Started process 53 to run task
[2022-11-21T10:40:36.292+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'python_dependences', 'task3', 'scheduled__2022-11-19T00:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/dag_test_with_python_dependence.py', '--cfg-path', '/tmp/tmp88ttw9xo']
[2022-11-21T10:40:36.360+0000] {standard_task_runner.py:83} INFO - Job 23: Subtask task3
[2022-11-21T10:40:37.091+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/configuration.py:545: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  option = self._get_environment_variables(deprecated_key, deprecated_section, key, section)

[2022-11-21T10:40:57.637+0000] {task_command.py:376} INFO - Running <TaskInstance: python_dependences.task3 scheduled__2022-11-19T00:00:00+00:00 [running]> on host f05724efbbd1
[2022-11-21T10:40:58.900+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=klein
AIRFLOW_CTX_DAG_ID=python_dependences
AIRFLOW_CTX_TASK_ID=task3
AIRFLOW_CTX_EXECUTION_DATE=2022-11-19T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-11-19T00:00:00+00:00
[2022-11-21T10:41:08.823+0000] {__init__.py:492} WARNING - Matplotlib created a temporary config/cache directory at /tmp/matplotlib-pthw_jlp because the default path (/home/airflow/.cache/matplotlib) is not a writable directory; it is highly recommended to set the MPLCONFIGDIR environment variable to a writable directory, in particular to speed up the import of Matplotlib and to better support multiprocessing.
[2022-11-21T10:41:11.991+0000] {font_manager.py:1443} INFO - generated new fontManager
[2022-11-21T10:41:34.479+0000] {logging_mixin.py:120} INFO - the version of yellowbrick==1.5
[2022-11-21T10:41:34.511+0000] {python.py:177} INFO - Done. Returned value was: None
[2022-11-21T10:41:34.864+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=python_dependences, task_id=task3, execution_date=20221119T000000, start_date=20221121T104035, end_date=20221121T104134
[2022-11-21T10:41:36.511+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-11-21T10:41:36.767+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
